guides/source/security.md

```diff
@@ -1170,7 +1170,7 @@ Rails.application.config.content_security_policy do |policy|
   policy.script_src :self, :https
 end
 
-Rails.application.config.content_security_policy_nonce_generator = -> request { SecureRandom.base64(16) }
+Rails.application.config.content_security_policy_nonce_generator = -> request { request.session.id.to_s }
```

Comment:
There are some security considerations to take into account when using `session.id`.
See: https://github.com/rails/rails/pull/43227#discussion_r711685813
I think we'd need to add those lines of caution and explain why this is needed for import maps.

Reply:
Updated to much more thoroughly describe the difference. After rereading the linked PR a bunch more times, I think I have a better understanding of the issue as well: I _think_ the reference to turbo/import map in the linked PR is about the necessity to use nonces (vs unsafe-inine) and the actual change was made because of the incompatibility with ETags (since the nonce was always invalidating the cache)